src/metpy/xarray.py

```diff
@@ -676,6 +682,22 @@ def parse_cf(self, varname=None, coordinates=None):
 
         var = self._dataset[varname]
 
+        # Check for crs conflict
```

Comment:
Don't want to slow getting this in, but should we add test(s) for this? Could scribble this down to go in during RC testing period if preferred.

Reply:
I went ahead and added those tests. I also got lost trying to silence the 131 warnings we have when running the tests, but that will have to wait for another PR (for 1.0 final).